### PR TITLE
fix periods in alerts api

### PR DESCRIPTION
**Pull request by BrandonArp**

### CODE REVIEW

File: app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java [hunk 1]
```diff
@@ -239,7 +239,7 @@ private Alert convertFromEbeanAlert(final models.ebean.Alert ebeanAlert) {
                 .setMetric(ebeanAlert.getMetric())
                 .setName(ebeanAlert.getName())
                 .setOperator(ebeanAlert.getOperator())
-                .setPeriod(Period.seconds(ebeanAlert.getPeriod()))
+                .setPeriod(Period.seconds(ebeanAlert.getPeriod()).normalizedStandard())
```
Comment (vjkoskela) [inline app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java]: Great catch!

**Comment by vjkoskela** [comment]
Looks like a failed test:

```

[error] Failed: Total 70, Failed 2, Errors 0, Passed 68
[error] Failed tests:
[error] 	com.arpnetworking.metrics.portal.alerts.impl.DatabaseAlertRepositoryTest
[error] 	controllers.AlertControllerTest
[error] (test:test) sbt.TestsFailedException: Tests unsuccessful
[error] Total time: 164 s, completed Sep 14, 2016 5:01:09 AM
```

File: app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java (hunk 1 above)
Comment (BrandonArp) [inline app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java]: Thanks.  Turns out that test failure was the same thing happening in the test code.  ) :

**Comment by BrandonArp** [comment]
Fixed the build.  Back to you.

**Pull request merged**